models/settings.js

```diff
@@ -0,0 +1,111 @@
+Settings = new Mongo.Collection('settings');
+
+Settings.attachSchema(new SimpleSchema({
+  strict: {
```

Comment:
Consider naming this to be more explicit. I.e. `strict` is ambiguous. More explicit names would be `memberInviteOnly`, `invitationOnly`, etc.